### ISSUE TITLE
Satisfaction ratings working not as intended

### ISSUE THREAD

**Issue by webpreparejordy** [freescout-helpdesk/freescout#1346]
Hello,

I have a big issue with the satisfaction ratings. When clients are clicking on for instance 'I am satisfied', it is marked in FreeScout as 'I am not satisfied' (Bad rating). How is this possible? I am using this code:

`<br>
Hoe beoordeel je mijn antwoord?<br>
<a href="{%ratings.great%}" style="color:#50bc1c;">Uitstekend</a> &nbsp;&nbsp; <a href="{%ratings.okay%}" style="color:#555555;">Neutraal</a> &nbsp;&nbsp; <a href="{%ratings.bad%}" style="color:#f10000;">Slecht</a><br>`

At least, this is my assumption. I asked those clients if they have clicked on an option. One of them has, but the other hasn't. Another weird issue is that when I created a new conversation by myself, it get's an 'Okay' rating.

Any ideas about this one?

Thanks,
Jordy

Edit:

Client confirmed ha hasn't even rated. Today another weird issue. Replied on a conversation, and this one suddenly get's the rating 'Okay'. Customer haven' clicked nor rated.

Edit 2:

I tried to replicate the issue and recorded it: https://www.loom.com/share/37e31568df5e41b78a9aa7cb4b3423f4

**Comment by webpreparejordy** [comment]
Additional comment:

Client confirmed ha hasn't even rated. Today another weird issue. Replied on a conversation, and this one suddenly get's the rating 'Okay'. Customer haven' clicked nor rated.

**Comment by densn93** [comment]
We have the same problem, in our case we only have "not good" ratings for some customers (every message).
One of them is a good friend of mine, so I asked why he always rates us with a bad rating - but he didn´t clicked on any rating.

Is it possible that some spam filters or mail softwares check the urls? 
If so, the last url in our e-mail is the "not good" rating and this would explain why it´s always the bad rating (because it checks this links at last)

**Comment by freescout-helpdesk** [comment]
Try to enable access log on your server and check what is opening URLs like '/feedback/8599/record/'

**Comment by webpreparejordy** [comment]
Ok, I was able to reproduce it and get access to the logs. What I did:

1. Created a new conversation in FreeScout and selecting an existing customer (email address is known in FreeScout)
2. Placed some content and sent the email to the email address jordy.boutier@kpnmail.nl
3. This conversation gets an 'Okay' rating

I found out that at the moment when I saw the Okay Rating, some logs popped up:

`185.107.232.169 - - [02/Aug/2021:10:48:53 +0200] "HEAD /feedback/588/record/JDJ5JDEwJFhDYWEuTjlLREFNL1M3Zm1TNVpVMGVkUWZ6YmVpTDNBdVJETzJPNHRrREJYZmdlVEM0TDVh/1 HTTP/1.1" 200 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36 Ecta/1.0"
185.107.232.169 - - [02/Aug/2021:10:48:53 +0200] "HEAD /feedback/588/record/JDJ5JDEwJFhDYWEuTjlLREFNL1M3Zm1TNVpVMGVkUWZ6YmVpTDNBdVJETzJPNHRrREJYZmdlVEM0TDVh/3 HTTP/1.1" 200 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36 Ecta/1.0"
185.107.232.169 - - [02/Aug/2021:10:48:54 +0200] "HEAD /feedback/588/record/JDJ5JDEwJFhDYWEuTjlLREFNL1M3Zm1TNVpVMGVkUWZ6YmVpTDNBdVJETzJPNHRrREJYZmdlVEM0TDVh/2 HTTP/1.1" 200 0 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36 Ecta/1.0"
    `
Hope this helps to solve this matter.

Thanks,
Jordy
